packages/manager/apps/pci-savings-plan/src/components/CreatePlanForm/CreatePlanForm.tsx

```diff
@@ -220,6 +219,19 @@ const CreatePlanForm: FC<CreatePlanFormProps> = ({
   const activeInstance = filteredTechnicalInfo?.find(
     (item) => item.name === technicalModel,
   );
+  const minDate = useMemo(() => {
+    const date = new Date();
+    date.setDate(date.getDate() + 1);
+    return date;
+  }, []);
+
+  const maxDate = useMemo(() => {
```

Comment:
it could be nice to reflect the date value in the names, what do you think? 
it would allow us to know the date value at a glance, without having to search for it in the code